.github/workflows/consumers.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-         os: ${{ fromJson(needs.get-oses.outputs.latest) }}
+         os: [ 'ubuntu-25.04' ]
```

Comment:
I guess github workers only have fedora and ubuntu, so once youo fixed it to ubuntu, it is disable for fedora?
That being said, is there any reason to use a hard-coded version of ubuntu?